### ISSUE TITLE
Can't download extension

### ISSUE THREAD

**Issue by finelystyles** [mozilla/multi-account-containers#1933]
- Multi-Account Containers Version: N/A
- Operating System + Version: Windows 10 1909
- Firefox Version: 84.0.1
- Other installed Add-ons + Version + Enabled/Disabled-Status:
Auto Tab Discard
Bitwarden Password Manager
ClearURLs
LocalCDN
Privacy Possum
Terms of Service; Didn't Read
uBlock Origin
Youtube Audio
(All of them are enabled)

### Actual behavior
I have previously used this addon and it's worked just fine, but a few weeks ago I noticed the tabs weren't opening in their assigned containers, so I uninstalled it. Now, when I try to install it, it gives me the error message: "Installation aborted because the add-on appears to be corrupt."

**Comment by miku86** [comment]
Hey @finelystyles,

do you use a custom `user.js` in your profile?

I can reproduce this behaviour on Firefox 84.0.1. All extensions disabled.

I have a default profile with a custom `user.js`. I get the same error as the issue author.

When I create a new profile, I can install the extension. If I then add my `user.js` to the profile folder and start this profile, it breaks.

The interesting part is, that I can make it work again, if I delete `user.js` and `prefs.js`. So in my case, there seems to be a setting in `user.js` that breaks the container extension installation.

Here is [my user.js](https://github.com/miku86/dotfiles/blob/master/firefox/user.js), maybe someone has an idea about the problematic setting.

Thanks
Michael

**Comment by miku86** [comment]
Got some new insights.

So I removed `prefs.js` and `user.js` from my profile.
Then I added the extension - this worked.
I re-added `prefs.js` and `user.js` to my profile.

The profile works and `Firefox Multi-Account Containers` shows up in the Add-ons section.
**But**: it doesn't show up in the address bar and it is also not in `Customize`.

When I go to `about:debugging#/runtime/this-firefox`, I can see the extension.
**But**: It doesn't have an `Internal UUID` and `Manifest URL` like every other add-on has.
When I click on `Inspect` it displays:

```
Error
Cannot connect to the debug target. See error details below:
TypeError: target is null.
```

My next step is to debug my `user.js` to see if there is a problematic setting.
